src/App.vue

```diff
@@ -51,30 +53,38 @@ export default {
 	},
 	async mounted() {
 		// Upcoming events
-		const today = EventUtil.getSecondsSince(Date.now());
+		const today = dayjs();
+		const nextMonth = dayjs().month(today.month() + 1);
 		const upcomingEvents = await EventUtil.fetchCalendarEvents(
 			UserUtil.getUserName(),
 			"personal",
-			today
+			today.unix(),
+			nextMonth.unix()
 		);
 
-		this.upcomingEvents = this.getNextFiveEvents(upcomingEvents);
+		this.upcomingEvents = this.getNextFiveNonAllDayEvents(upcomingEvents);
+
 		const bookingNews = await NewsUtil.fetchBookingNews();
 		const vaMessages = await NewsUtil.fetchVaMessages();
 		this.upcomingNews = this.upcomingNews.concat(vaMessages, bookingNews);
 	},
 	methods: {
-		getNextFiveEvents: (events) =>
+		getNextFiveNonAllDayEvents: (events) =>
 			events
-				.sort(
-					(evt1, evt2) =>
-						new Date(evt1.dateStart).getTime() -
-						new Date(evt2.dateStart).getTime()
-				)
+				.filter((evt) => !evt.isAllDay)
+				.sort((evt1, evt2) => {
+					const c = new Date(evt1.dtstart);
+					const d = new Date(evt2.dtstart);
+					return c - d;
+				})
 				.slice(0, Math.min(5, events.length)),
 		getEventsByUniqueStartDate: (events) =>
```

Comment:
Something could be added in the future like a "No sessions in the next 5 days" message, since the user doesn't know that the app only shows the next 5 days events

Reply:
Fixed in latest commit